frontend/packages/core-api-client/src/types/model/Form.ts

```diff
@@ -24,3 +24,8 @@ export type FormWithRecord<T extends Record> = {
   form: FormDefinition;
   record: T;
 };
+
+export type FormWithRecords<T extends Record> = {
```

Comment:
`FormWithRecordList`, I think adding an `s` on the end of a type makes it difficult to distinguish from the singular at a glance.